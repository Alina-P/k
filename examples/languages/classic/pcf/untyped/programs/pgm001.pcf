(lambda addd . ((mu foo . lambda xx . lambda yy . cond (zero? xx) 0 (addd (foo (pred xx) yy) yy)) 7 9)) (mu foo . lambda xx . lambda yy . cond (zero? xx) yy (succ (foo (pred xx) yy)))


