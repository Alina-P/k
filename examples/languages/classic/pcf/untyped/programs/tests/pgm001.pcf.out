<result>
  63
</result>